test/trackler/problem_test.rb

```diff
@@ -82,6 +82,10 @@ def test_track_with_no_canonical_data
     assert_equal nil, problem.canonical_data_url
   end
 
+  def test_problem_with_no_metadata_yml
+    refute Trackler::Problem.new('no-metadata', FIXTURE_PATH).exists?
+  end
```

Comment:
Having x-api rely on fixture data in the trackler gem is **really** messed\* up.

> I want x-common's travis build to fail in that case.

OK. That's something to I can work on.

\* expletive substituted.